traitsui/tests/editors/test_instance_editor.py

```diff
@@ -146,10 +166,10 @@ def test_custom_editor_with_selection(self):
             # test that the current object is None
             self.assertIsNone(obj.inst)
 
-            # test that the displayed text is correct
+            # test that the displayed text is empty
             instance = tester.find_by_name(ui, "inst")
             text = instance.inspect(SelectedText())
-            self.assertEqual(text, obj.inst_list[0].name)
+            self.assertEqual(text, '')
```

Comment:
i guess this is the right behavior - and previously, we were testing/checking for the buggy/wrong behavior.

Reply:
Yeah, exactly.  The old test I was more concerned with UITester support than behavior of the editor 😅.  I had basically manually tested a UI and then wrote UITester test that matched what I saw.